src/ex_run.rs

```diff
@@ -67,7 +67,7 @@ pub fn run_ex_all_tcs(ex_name: &str) -> Result<()> {
 }
 
 pub fn run_ex(ex_name: &str, tc: &str) -> Result<()> {
-    let tc = toolchain::parse_toolchain(tc)?;
+    let tc = tc.parse::<toolchain::Toolchain>()?;
```

Comment:
```rust
let tc = tc.parse::<toolchain::Toolchain>()?;
```
```rust
let tc: toolchain::Toolchain = tc.parse()?;
```

personally, i find both of these equally understandable

```rust
let tc = tc.parse()?;
```

this one is a harder sell for me just because it takes some thinking and searching to find out what type the new `tc` is, so i personally don't really do this, but am not completely opposed